core-bundle/src/Framework/Adapter.php

```diff
@@ -20,6 +20,8 @@
  * @mixin T
  *
  * @internal Do not use this class in your code; use ContaoFramework::getAdapter() instead
+ * @template T
+ * @mixin T
```

Comment:
This is redundant now and needs to be removed. 🙈

Reply:
Resolved with 43e9d1d